acareeg/eegip.py

```diff
@@ -171,53 +175,122 @@ def process_events_london_resting_state(raw, age, tmax=1):
 
         if len(rst0) == 0 and len(rst1) == 0:
             return None
```

Comment:
as i understand lines 168-177:
if raw.annotations['description] says Rst0, append rest0 with the onset latency of raw.annotations?
same goes for Rst1 lines 173-174

176: if there are not Rst0 or Rst1 tags, return none. i.e. participant didnt receive resting.